src/main/java/io/kestra/plugin/pulsar/Reader.java

```diff
@@ -43,16 +43,16 @@ public class Reader extends AbstractReader {
     @Schema(
         title = "The initial reader positioning can be set at specific timestamp by providing total rollback duration.",
         description = "So, broker can find a latest message that was published before given duration. eg: " +
-            "rollbackDuration in minute = 5 suggests broker to find message which was published 5 mins back and " +
+            "rollbackDuration in minute = 5 suggests broker to find message which was published 5 minutes back, and " +
```

Comment:
```suggestion
            "`rollbackDuration` in minutes; setting it to 5 indicates that broker should find messages published 5 minutes in the past, and " +
```

Reply:
Corrected this message, but made some improvements further.